packages/mui-material/src/styles/createTypography.d.ts

```diff
@@ -27,28 +28,6 @@ export interface FontStyle {
   htmlFontSize: number;
 }
 
-export type NormalCssProperties = CSS.Properties<number | string>;
```

Comment:
@mui/core Am I right in thinking this shouldn't be duplicated?

Reply:
I seems right to me as well, although I don't have the context as to why it is duplicated in the first place.

@mnajdova do you know?

For reference, the duplication is here: https://github.com/mui/material-ui/blob/51663eba0e020daa6b494c31391560f0dd107489/packages/mui-material/src/styles/createMixins.d.ts

Reply:
Tl;dr yes, it should be safe for these to be removed

From what I remember these were initially copied from @mui/styles to make the JSS <-> Emotion interoperability work. They may have been accidently copied in two locations, so it's safe to be removed from here. Eventually we could likely drop these, especially now that the @mui/styles package is deprecated.

Reply:
Thanks!